Holo-Host/hp-admin
# 2020 06 16/adjusting polling

### Updates :
- uses request options from useQuery to dynamically stop/start polling

## To Test:
 - spin up a running conductor, sim2h and single agent
 - wait for app to load in the browser and successfully connect to holochain with regular polling to 3 endpoints (each set at 5000ms)
 - turn off the conductor
 - wait for 3-4 more network calls to be made and then notice a redirect to the home page with a modal stating that 'Your Conductor is currently unreachable'.
  - notice that all calls expect the LedgerQuery have stopped polling

...to test that it can reconnect:
  - turn your conductor back on
  - notice the connection to holochain is re-established
  - notice that all network calls complete successfully and that polling is reset for appropriate endpoints
